src/mumble/Log.cpp

```diff
@@ -571,6 +576,17 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 		}
 	}
 
+	// TTS No Author
+	if (g.s.bTTSNoAuthor){
+		int authorNameLen = 0;
+		authorNameLen = plain.indexOf(QLatin1String(": "), 0) + 2;
+		plain.remove(9, authorNameLen-9);
+	}
+
+	// TTS Brief Messages
+	if (g.s.bTTSBriefMessages && plain.contains(QRegExp(QLatin1String("^\\(Channel\\)"))))
```

Comment:
there is no need for a regex here. You can check `plain.startsWith("(Channel)")`

Reply:
What do you think of using similar logic as my suggestion above? Simply remove anything between (and including) the first instance of "(" and ")"